### ISSUE TITLE
Extend benchmark_trainers

### ISSUE THREAD

**Issue by accosmin** [accosmin/nanocv#85]
* save the train & validation loss values/errors as csv
* plot & compare their evolution with the number of iterations & seconds
* extend the csv produced by trainer_state with an additional column as training time in seconds

**Comment by accosmin** [comment]
Also make the plotting scripts more user friendly (e.g. usage & input checks) and more modular/compact.

**Comment by accosmin** [comment]
Improve axis labels: always use epochs for Ox or seconds (if plotting by time), use loss or error for Oy.

**Comment by accosmin** [comment]
Thinner lines: it is very hard to zoom in and to distinguish the training loss from the validation loss.
